.travis.yml

```diff
@@ -1,16 +1,13 @@
 language: node_js
 sudo: false
 node_js:
-  - 0.12
-  - 4
-  - 5
-  - 6
+  - 7.8.0
 # NodeJS v4 requires gcc 4.8
 env:
   - NODE_ENV=travis CXX="g++-4.8" CC="gcc-4.8"
 matrix:
   allow_failures:
-    - node_js: 6
+    - node_js: 7.8.0
```

Comment:
just remove the allow_failures completely as it doesn't make sense to require only 7.8.0 and also let it fail.